app/app.go

```diff
@@ -838,11 +839,40 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 }
 
 // RegisterUpgradeHandlers returns upgrade handlers
-
+// RegisterUpgradeHandlers returns upgrade handlers
 func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
 	app.UpgradeKeeper.SetUpgradeHandler("v10",
 		func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-			// transfer module consensus version has been bumped to 2
+
+			// update ICA Host to catch missed msg
+			// enumerate all because it's easier to reason about
+			newIcaHostParams := icahosttypes.Params{
+				HostEnabled: true,
+				AllowMessages: []string{
+					sdk.MsgTypeURL(&banktypes.MsgSend{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgDelegate{}),
+					sdk.MsgTypeURL(&stakingtypes.MsgUndelegate{}), // this was missed last time
```

Comment:
```suggestion
					sdk.MsgTypeURL(&stakingtypes.MsgUndelegate{}), // this was missed last time
					sdk.MsgTypeURL(&ibctransfertypes.MsgTransfer{}),
```

Would it be possible to allow ICA IBC transfers as well?

For reference, these are the allow messages on the cosmos hub

```
allow_messages:
- /cosmos.authz.v1beta1.MsgExec
- /cosmos.authz.v1beta1.MsgGrant
- /cosmos.authz.v1beta1.MsgRevoke
- /cosmos.bank.v1beta1.MsgSend
- /cosmos.bank.v1beta1.MsgMultiSend
- /cosmos.distribution.v1beta1.MsgSetWithdrawAddress
- /cosmos.distribution.v1beta1.MsgWithdrawValidatorCommission
- /cosmos.distribution.v1beta1.MsgFundCommunityPool
- /cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward
- /cosmos.feegrant.v1beta1.MsgGrantAllowance
- /cosmos.feegrant.v1beta1.MsgRevokeAllowance
- /cosmos.gov.v1beta1.MsgVoteWeighted
- /cosmos.gov.v1beta1.MsgSubmitProposal
- /cosmos.gov.v1beta1.MsgDeposit
- /cosmos.gov.v1beta1.MsgVote
- /cosmos.staking.v1beta1.MsgEditValidator
- /cosmos.staking.v1beta1.MsgDelegate
- /cosmos.staking.v1beta1.MsgUndelegate
- /cosmos.staking.v1beta1.MsgBeginRedelegate
- /cosmos.staking.v1beta1.MsgCreateValidator
- /cosmos.vesting.v1beta1.MsgCreateVestingAccount
- /ibc.applications.transfer.v1.MsgTransfer
- /tendermint.liquidity.v1beta1.MsgCreatePool
- /tendermint.liquidity.v1beta1.MsgSwapWithinBatch
- /tendermint.liquidity.v1beta1.MsgDepositWithinBatch
- /tendermint.liquidity.v1beta1.MsgWithdrawWithinBatch
```

Reply:
Sorry didn't see this - what's the use case for IBC transfers?

Reply:
For ref, IBC transfers have been added - https://github.com/CosmosContracts/juno/pull/290/commits/f2f9de4467be8ce64f86eded128b87c9364fd39a